pkg/yurtctl/util/kubernetes/util.go

```diff
@@ -359,6 +358,52 @@ func CreateCRDFromYaml(clientset *kubernetes.Clientset, yurtAppManagerClient dyn
 	return nil
 }
 
+func DeleteCRD(clientset *kubernetes.Clientset, yurtAppManagerClientSet dynamic.Interface, res string, name string) error {
+	var err error
+	decoder := yamlutil.NewYAMLOrJSONDecoder(bytes.NewReader([]byte(constants.YurtAppManagerNodePool)), 10000)
```

Comment:
Why use `constants.YurtAppManagerNodePool`? As this func `DeleteCRD` is also called when delete 
 `UnitedDeploymentcrd` in func `removeYurtAppManager`, the parameter also can be `constants.YurtAppManagerUnitedDeployment`.